server/admin_management/tests/constants.py

```diff
@@ -65,7 +65,7 @@
 # -------------------- test forest client data ---------------------- #
 TEST_NON_EXIST_FOREST_CLIENT_NUMBER = "99999999"
 TEST_INVALID_FOREST_CLIENT_NUMBER = "12345"
-TEST_INACTIVE_FOREST_CLIENT_NUMBER = "00000002"
+TEST_INACTIVE_FOREST_CLIENT_NUMBER = "00000003"
```

Comment:
Can ignore this. Forest Client API probably refreshes data and causing FAM's pipeline failed on tests due to this number used on testing. Just a quick fix to avoid for now for this PR.